schemas/vulnerability.yml

```diff
@@ -0,0 +1,149 @@
+
+        Environmental scores cover an assessment for any modified Base metrics, confidentiality, integrity, and availability requirements. For example (ex. https://www.first.org/cvss/specification-document)
+
+      example: 5.5, 6.6
+
+    - name: score.version
+      level: extended
+      type: long
+      short: CVSS version.
+      description: >
+        The National Vulnerability Database (NVD) provides qualitative severity rankings of "Low", "Medium", and "High" for CVSS v2.0 base score ranges in addition to the severity ratings for CVSS v3.0 as they are defined in the CVSS v3.0 specification.
+
+        CVSS is owned and managed by FIRST.Org, Inc. (FIRST), a US-based non-profit organization, whose mission is to help computer security incident response teams across the world. For example (ex. https://nvd.nist.gov/vuln-metrics/cvss)
+
+      example: 2.0, 3.0
+
+    - name: category
+      level: extended
+      type: keyword
+      short: Category of a vulnerabilty.
+      description: >
+        The type of system or architecture that the vulnerability affects. These may be platform-specific (for example, Debian or SUSE) or general (for example, Database or Firewall). For example (ex. https://qualysguard.qualys.com/qwebhelp/fo_portal/knowledgebase/vulnerability_categories.htm)
+
+      example: AIX, Firewall, Malware, Office Application, Solaris, Windows
+
+    - name: description
+      level: extended
+      type: keyword
+      short: Description of the vulnerability.
+      description: >
+        The description of the vulnerability that provides additional context of the vulnerability. For example (ex. https://cve.mitre.org/about/faqs.html#cve_entry_descriptions_created)
+
+      example: In Keybase before 2.12.6 on macOS, the move RPC to the Helper was susceptible to time-to-check-time-to-use bugs and would also allow one user of the system (who didn't have root access) to tamper with another's installs.
+
+    - name: scanner.id
+      level: extended
+      type: keyword
+      short: ID of the vulnerability.
+      description: >
+        The identification (ID) is the number portion of a vulnerability entry. It includes a unique identification number for the vulnerability. For example (ex. https://cve.mitre.org/about/faqs.html#what_is_cve_id)
+
+      example: CVE-2019-00001, CVE-2019-00002, QID: 38739
+
+    - name: scanner.vendor
+      level: extended
+      type: keyword
+      short: Name of the scanner vendor.
+      description: >
+        The name of the vulnerability scanner vendor.
+
+      example: Tenable, Qualys
+
+    - name: severity
+      level: extended
+      type: keyword
+      short: Severity of the vulnerability.
+      description: >
+        The severity of the vulnerability can help with metrics and internal prioritization regarding remediation. For example (ex. https://nvd.nist.gov/vuln-metrics/cvss)
+
+      example: Critical, High, Medium, Low, None
+
+    - name: status
```

Comment:
Do vulnerability scanners keep track of past runs, and can they report a status of "re-opened"? Or is this more of a concept that you'd expect to see in a UI that knows about that past history of scans?

I'm wondering if it's a situation like the first/last seen timestamps we discussed, last time around :-)